dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,376 @@
+                    "title": "How it works",
+                    "text": [
+                        "How does Dataland work?"
+                    ],
+                    "cards": [
+                        {
+                            "title": "Search",
+                            "text": "Utilize the search feature to find specific companies within the platform."
+                        },
+                        {
+                            "title": "Request company’s inclusion",
+                            "text": "Request company’s inclusion in case you can’t find the company you are looking for."
+                        },
+                        {
+                            "title": "Request framework data",
+                            "text": "Create or join a campaign to get the missing data."
+                        },
+                        {
+                            "title": "Download",
+                            "text": "Download either through the platform or through API."
+                        }
+                    ]
+                },
+                {
+                    "title": "Claim",
+                    "text": [
+                        "In Dataland, ",
+                        "you",
+                        " have the power."
+                    ]
+                },
+                {
+                    "title": "Campaigns",
+                    "text": [
+                        "Create or join a campaign",
+                        " to get the missing data."
+                    ],
+                    "cards": [
+                        {
+                            "icon": "/static/icons/Crop-growth.svg",
+                            "title": "SFDR",
+                            "text": "101",
+                            "date": "14th of August"
+                        },
+                        {
+                            "icon": "/static/icons/Diagram--reference.svg",
+                            "title": "EU Taxonomy",
+                            "text": "269"
+                        },
+                        {
+                            "icon": "/static/icons/Dashboard--reference.svg",
+                            "title": "LksG",
+                            "text": "101",
+                            "date": "14th of August"
+                        },
+                        {
+                            "icon": "/static/icons/Explore.svg",
+                            "title": "Pathway to Paris",
+                            "text": "101",
+                            "date": "Starting soon!"
+                        }
+                    ]
+                },
+                {
+                    "title": "Come together",
+                    "text": [
+                        "Collaborate on datasets ",
+                        "to share the costs."
+                    ],
+                    "image": [
+                        "Join now!"
```

Comment:
Why is there a text here?  Other sections have paths to image files here.

Reply:
An image will come, for now I placed a button with this text. This component (TheComeTogether) won't go live anyway

Reply:
Ah ok 👍 .

But I think we should not have section-info in the `content-json` file for components, that we don't display yet.

Can we keep it in another branch or on your IDEs shelve maybe instead merging it to main?